uaclient/api/api.py

```diff
@@ -18,6 +18,7 @@
     "u.pro.attach.magic.initiate.v1",
     "u.pro.attach.magic.wait.v1",
     "u.pro.attach.magic.revoke.v1",
+    "u.pro.auto_attach.with_short_retry.v1",
```

Comment:
Moving the endpoint naming discussion to an inline-comment so that it can have it's own comment thread. @lucasmoura @CalvoM 

We already have an endpoint namespace for `attach` so we should reuse that.
I also think we will likely have the individual steps of auto-attach available as endpoints in the future, so we should be clear this endpoint is doing the whole auto-attach process.
The number of retries should probably be an optional parameter so I don't think "retry" should be in the name.

With all of that in mind, I propose: `u.pro.attach.auto.full_auto_attach.v1`

What do you think?

Reply:
+1 on the endpoint name

Reply:
Makes sense. I saw the `attach` as a possible namespace to use, you have convinced me to change to it.